spec/support/manager.cr

```diff
@@ -2,5 +2,6 @@ class Manager < BaseModel
   table do
     column name : String
     has_many employees : Employee
+    has_many business_sales : BusinessSale, through: :employees
```

Comment:
In #196 I had also run in to an issue of a has_many through a has_many through.. Just to ensure that's also covered, can we maybe add in another `has_many customers : Customer, through: :business_sales`? or something along those lines.

Reply:
@jwoertink There are definitely still bugs when doing a has_many through a has_many through but I don't have a good solution for them and it would add a lot more changes to this PR which is already confusing enough. I updated the description to not mark that issue as fixed